### PR TITLE
LPS-202228 Change Configuration view for Blogs

### PR DESCRIPTION
**Pull request by jkappler**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-202228)

## Changes

Change the configuration for Blogs because it should be open in a new page instead of a modal.

## Notes

**You need to activate the LPS-197692 FF.**

## Test Scenario 1

1. Go to Blogs
2. Click on the kebab menu in the top right
3. Navigate to the sections to check the behaviour and styles

https://github.com/liferay-echo/liferay-portal/assets/93203423/53111ec4-ccc4-43ad-a77b-ccdd690b47ed

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by jkappler** [comment]
@BarbaraCabrera, @sandrodw3, all good from my side, don't know if you want to review it again before sending to Lima team ❤️

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/configuration_browser.jsp
Comment (sandrodw3) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/configuration_browser.jsp:1]: This file is called `configuration_browser` and the one in Journal `configuration-browse`. This is a bit weird, right?

**Review by sandrodw3 — COMMENTED** [review]
Hey @BarbaraCabrera very nice job, this is working very well! 🥳 Just left a small comment, and I also have some additional thoughts:

- Just mentioning, the change to the RSS taglib is delicate, that taglib is being used in a lot of places in portal, and we are changing the display style for all of them. Could you confirm with Design that they know we are applying this to the whole portal? 😛
- A tip for future PRs, in my opinion, changing the same file in different commits makes the review more difficult. From my point of view, adding new lines is not a problem, but removing something I added in a previous commit is something I always try to avoid 😄

Adding "Changes Required" label in case you want to rename the file I mentioned.

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/configuration_browser.jsp
Comment (jkappler) [reply modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/configuration_browser.jsp:1]: Didn't catch this, thanks @sandrodw3!

**Comment by BarbaraCabrera** [comment]
See: https://github.com/liferay-lima/liferay-portal/pull/4950

**Pull request closed without merging**